esti/migrate_test.go

```diff
@@ -584,9 +589,19 @@ func testPreMigrateGraveler(t *testing.T) {
 
 	uploadFiles(t, ctx, repo, "pre_migrate_branch", "b/foo/")
 
+	// Add branch protection rules for migration
+	createBpResp, err := client.CreateBranchProtectionRuleWithResponse(ctx, repo, api.CreateBranchProtectionRuleJSONRequestBody{
+		Pattern: "*SomePattern",
+	})
+	require.Equal(t, http.StatusNoContent, createBpResp.StatusCode())
+
+	getBpResp, err := client.GetBranchProtectionRulesWithResponse(ctx, repo)
+	require.Equal(t, http.StatusOK, getBpResp.StatusCode())
```

Comment:
usually I prefer to match JSON200 != nil - it will verify the response content also match the expectation